people.html

```diff
@@ -0,0 +1,192 @@
+            <a class="navbar-brand" href="index.html">
+				Soteria Collaborators
+			</a>
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-link" href="demo.html">Demo</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="people.html">People</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="wiki.html">Wiki</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <main class="container style-fix">
+        <h1>Software</h1>
+        <div class="container">
+            <div class="row row-cols-4">
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/Ismael.jpg" alt="Place holder">
+                    <p>Ismael <br> Core <br> team leader</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/avery.jpg" alt="Avery">
+                    <p>Avery <br> Core</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p> Khoa <br> Core</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p>Chris <br> Core <br> VR </p>
+                </div>
+            </div>
+            <div style="background-color:black; border: 1px solid;"></div>
+            <div class="row row-cols-4">
+                <!--col-lg-3 means that on large scrrens, colmn will take up fourth the length (12/3 = 4).
+                col-md-4 means that on medium screens, colmn will take up a third of the length (12/4 = 3).  
+                col-sm-6 means that on small screens, colmn will take up half the of the length (12/6 = 2).-->
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/Julia.jpeg" alt="Place holder">
+                    <p>Julia <br> VR</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/Sebastian.jpg" alt="Sebastian">
+                    <p>Sebastian <br> Software-Website</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p>Sahil <br> Software-Website</p>
+                </div>
+                <div class="col-lg-3 col-md-4 col-sm-6">
+                    <img src="assets/images/circuit_board.png" alt="Place holder">
+                    <p>Son <br>Software-Website</p>
+                </div>
+            </div>
+            <div style="background-color:black; border: 1px solid;"></div>
```

Comment:
I think its better to minimize usage of inline CSS, but this is small so I am open to keeping it. If we wanted to use bootstrap we could use something like `<hr></hr>` or `<div class="divider"></div>`